frontend/include/chpl/util/filesystem.h

```diff
@@ -20,9 +20,8 @@
 #ifndef CHPL_UTIL_FILESYSTEM_H
 #define CHPL_UTIL_FILESYSTEM_H
 
-#include "chpl/framework/ErrorMessage.h"
-
 #include <string>
+#include <system_error>
```

Comment:
Does this need to be in the header, or should it be in the source file(s)?

Reply:
It should be in the header as it provides `std::error_code` which is the return type of `ensureDirExists` declared in this header.